dataland-frontend/src/components/forms/CreateEuTaxonomyForFinancials.vue

```diff
@@ -55,10 +55,11 @@
               </div>
               <FormKit type="group" name="data" label="data" validation-label="data" validation="required">
                 <UploadReports
+                  name="UploadReports"
                   ref="UploadReports"
-                  :editMode="editMode"
+                  :isEuTaxonomy="true"
                   :referencedReportsForPrefill="templateDataset?.referencedReports"
-                  @referenceableReportNamesChanged="handleChangeOfReferenceableReportNames"
+                  @referenceUpdated="handleChangeOfReferenceableReportNames"
```

Comment:
I'm still not 100% happy with the name.  A  "reference" is a hash in our code base.
And it is not a hash that is updated in this event, it is one (or more) report.
So maybe "reportsUpdated"? With an "s", because it might be several ones?

Reply:
Pushed latest. Changed to `reportsChanged`. Done.

Reply:
I think you changed it to "reportsUpdated", but that is even better.